modules/exploits/windows/local/ntusermndragover.rb

```diff
@@ -95,6 +96,9 @@ def check
       return CheckCode::Safe
     end
 
+    path = expand_path('%WINDIR%\\system32\\win32k.sys')
+    major, minor, build, revision, brand = file_version(path)
+    return CheckCode::Safe if revision >= 24387
```

Comment:
I'm a bit hesitant to just rely on the `revision` number, seeing as I don't know if this increments across Windows versions equally or if it gets reset when they release a new version of Windows. I did test and confirm that this is the right version for Windows 7 SP1 though and that you cannot apply the patch to Windows 7 SP0.